src/Data/NodeResolver.php

```diff
@@ -60,7 +60,10 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 				],
 				true
 			) ) {
-				throw new UserError( __( 'Cannot return a resource for an external URI', 'wp-graphql' ) );
+				graphql_debug( __( 'Cannot return a resource for an external URI', 'wp-graphql' ), [
```

Comment:
One thing to note is this isn't an external uri, it's a 404 - it'd be good to update the error message with that in mind